amp.js

```diff
@@ -66,6 +66,7 @@ createTask('get-zindex', 'getZindex');
 createTask('integration');
 createTask('lint');
 createTask('make-extension', 'makeExtension');
+createTask('make-extension-e2e', 'runMakeExtensionTests', 'make-extension');
```

Comment:
I don't think this should be a task on its own. The tasks listed here are all mostly independent and unrelated from each other. Instead, this should be a flag to the `make-extension` task. (See my other comment for details.)

Reply:
Added `--test` and `--cleanup`.

Reply:
SGTM!